packages/components/text/src/Text.tsx

```diff
@@ -107,6 +104,7 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
     ...pressable?.props,
     numberOfLines: truncate || !wrap ? 1 : 0,
     onKeyDown: Platform.OS === (('win32' as any) || 'windows') ? onKeyDown : undefined,
+    onMouseEnter: Platform.OS === (('win32' as any) || 'windows') ? pressable.props.onMouseEnter : undefined,
```

Comment:
Could we combine these into an object we spread and then have a comment like "These callbacks are not implemented on iOS/macOS, and cause Redboxes if passed in. Limit to only windows/win32 for now".

Also not sure if they cause issues on Android but I think it's fine to filter to only windows.

Reply:
Also, `onMouseLeave`